scipy/stats/stats.py

```diff
@@ -5718,3 +5720,179 @@ def rankdata(a, method='average'):
 
     # average method
     return .5 * (count[dense] + count[dense - 1] + 1)
+
+
+def rvs_ratio_unif(f, size=1, x1=None, x2=None, y2=None, shift=0, maxiter=None,
```

Comment:
Coming from an unrelated area I think it would work better if it was:
`rvs_ratio_unif(pdf, ...`

Reply:
`shift` isn't described well in the list of parameters.